ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -235,19 +246,23 @@ class FamilyReads extends React.PureComponent {
     familiesByGuid: PropTypes.object,
     individualsByGuid: PropTypes.object,
     igvSamplesByFamilySampleIndividual: PropTypes.object,
+    genesById: PropTypes.object,
   }
 
   state = {
     openFamily: null,
     sampleTypes: [],
     rnaReferences: [],
     minJunctionEndsVisible: 0,
+    locus: null,
   }
 
   showReads = familyGuid => sampleTypes => () => {
+    const { variant, projectsByGuid, familiesByGuid } = this.props
     this.setState({
       openFamily: familyGuid,
       sampleTypes,
+      locus: variant && getVariantLocus(variant, projectsByGuid[familiesByGuid[familyGuid].projectGuid]),
```

Comment:
not crucial, but it might be helpful/ more readable to write a helper function like `this.getProjectForFamily(familyGuid)` that is used  both here and in the main render method